airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/utils/object_helpers.py

```diff
@@ -28,3 +30,17 @@ def f(*args, **kwargs):
         return to_call(*copy.deepcopy(args), **copy.deepcopy(kwargs))
 
     return f
+
+
+def to_json_sanitized_dict(pydantic_model_obj: BaseModel) -> dict:
+    """A helper function to convert a pydantic model to a sanitized dict.
+
+    Without this pydantic dictionary may contain values that are not JSON serializable.
+
+    Args:
+        pydantic_model_obj (BaseModel): a pydantic model
+
+    Returns:
+        dict: a sanitized dictionary
+    """
+    return json.loads(pydantic_model_obj.json())
```

Comment:
I feel like this is something that should be exposed at the model level as a class method. I guess it's hard to move it there because models are generated.

Reply:
Exactly, however I could turn it into a mixin. Let me know if you think thats a better choice. Im on the fence